tests/foreman/cli/test_satellitesync.py

```diff
@@ -81,6 +81,23 @@ def export_import_cleanup_module(target_sat, module_org):
     )
 
 
+@pytest.fixture(scope='class')
+def docker_repo(module_target_sat, module_org):
+    product = make_product({'organization-id': module_org.id})
+    repo = make_repository(
```

Comment:
we already have some quay container repo definition specified in `settings.container_repo`. Is this file something we could use here instead of hardcoding stuff?

Reply:
there is a pulp_container related issue for this repo: https://github.com/pulp/pulp_container/issues/1227
and BZ: https://bugzilla.redhat.com/show_bug.cgi?id=2181254

Though it was already addressed in upstream, not sure if it makes into 6.13.